en/docs/administer/key-managers/configure-global-key-manager.md

```diff
@@ -0,0 +1,124 @@
+
+    [![Sample App Creation]({{base_path}}/assets/img/administer/global-keymanager/sample-app-creation.png)]({{base_path}}/assets/img/administer/global-keymanager/sample-app-creation.png)
+
+12. Click **Production Keys** and Select the Global Key Manager (IS-GKM) which was created from the list of key managers displayed.
+
+    [![Select Global Key Manager]({{base_path}}/assets/img/administer/global-keymanager/select-global-key-manager.png)]({{base_path}}/assets/img/administer/global-keymanager/select-global-key-manager.png)
+
+13. Click **GENERATE KEYS**.
+
+14. Click **GENERATE ACCESS TOKEN** to generate an application access token. Make sure to copy the generated JWT access token that appears so that you can use it in the future.
+
+15. Go to the Developer Portal landing page, select the SampleAPI and [Subscribe]({{base_path}}/consume/manage-subscription/subscribe-to-an-api/#subscribe-to-an-existing-application) to the SampleApp.
+
+    [![Subscribe SampleAPI to SampleApp]({{base_path}}/assets/img/administer/global-keymanager/subscribe-sample-api-to-sample-app.png)]({{base_path}}/assets/img/administer/global-keymanager/subscribe-sample-api-to-sample-app.png)
+
+16. [Try Out the API]({{base_path}}/consume/invoke-apis/invoke-apis-using-tools/invoke-an-api-using-the-integrated-api-console/) with the access token generated using the Global Key Manager.
+
+    A Successful response indicates that an API of the super tenant can be invoked using an access token generated for an application using the Global Key Manager.
+
+17. Click **SWITCH DEV PORTALS** which is at the top of the navigation bar.
+
+    [![Switch Developer Portal]({{base_path}}/assets/img/administer/global-keymanager/switch-dev-portal.png)]({{base_path}}/assets/img/administer/global-keymanager/switch-dev-portal.png)
+
+18. Select the tenant which was created in Step 8 from the list of tenants displayed.
+
+    [![Tenant List]({{base_path}}/assets/img/administer/global-keymanager/tenant-list.png)]({{base_path}}/assets/img/administer/global-keymanager/tenant-list.png)
+
+19. Select the SampleAPI and Subscribe to the SampleApp similar to Step 15.
+
+    !!! Note
+        The keys generated for an application using the Global Key Manager will be displayed across tenants. In this example scenario, the keys for the sampleApp were generated in the Super Tenant’s Developer Portal. These keys are reflected in the Production Keys of the IS-GKM tab of the sampleApp in the abc.com’s Developer Portal.
+
+20. Try Out the API with the same access token generated using the Global Key Manager which was used in Step 16.
+
+A Successful response indicates that the API of a tenant can be invoked using an access token generated for an application using the Global Key Manager. Note that the access token was generated in the super tenant’s Developer Portal, but an API in a different tenant was invoked using this access token.
```

Comment:
Shall we change 'S' in "Successful" to lower case?

Reply:
fixed